app/models/price_group.rb

```diff
@@ -17,7 +17,7 @@ class PriceGroup < ApplicationRecord
 
   default_scope -> { order(is_internal: :desc, display_order: :asc, name: :asc) }
 
-  before_destroy { throw :abort if global? }
+  before_destroy :throw_abort, if: :global?
```

Comment:
It's mainly just a personal preference., but I kind of prefer the block form for one-liners unless the method name aids understanding intention.

What's your opinion for why you pulled this up into a method?

Reply:
Ok, I get it now. You made it a method so you could skip it in the rake task.